### DISCUSSION TITLE
Uploading files on a  SharePoint / Office 365 Shared Libraries (Business or Education)

### DISCUSSION THREAD

**Discussion by thiagojcb** [abraunegg/onedrive#2291] [Q&A] [answered]
I'm using v2.4.23-1-gc10870a and having a problem to sync files in a sharedpoint library.
I did all the steps on the [SharePoint-Shared-Libraries.md](https://github.com/abraunegg/onedrive/blob/master/docs/SharePoint-Shared-Libraries.md) and it works up to the first synchronise. Then, if I add a file in my local directory, and I synchronise again (eg: `onedrive --confdir=".config/liquido_onedrive" --synchronize`)  I get the following error:

```
ERROR: The local file system returned an error with the following message:
  Error Message:    .config/liquido_onedrive/refresh_token: No such file or directory
```

The file exists though. 
```
$ ll .config/liquido_onedrive/
total 96
drwxrwxr-x  2 thiago thiago  4096 Jan 16 12:20 ./
drwx------ 56 thiago thiago  4096 Jan 16 10:00 ../
-rw-rw-r--  1 thiago thiago  1824 Jan 13 22:45 config
-rw-rw-r--  1 thiago thiago  1760 Jan 13 22:43 config~
-rw-------  1 thiago thiago  1824 Jan 13 22:45 .config.backup
-rw-------  1 thiago thiago    28 Jan 13 22:56 .config.hash
-rw-rw-r--  1 thiago thiago 32768 Jan 13 23:57 items-dryrun.sqlite3
-rw-r--r--  1 thiago thiago 32768 Jan 16 12:20 items.sqlite3
-rw-------  1 thiago thiago   882 Jan 16 12:06 refresh_token
```

I've tried deleting refresh_token and resync, but still get the same issue.

Thanks.

**Comment by abraunegg** [comment, marked as answer]
@thiagojcb 
Please re-read step 5 - https://github.com/abraunegg/onedrive/blob/master/docs/SharePoint-Shared-Libraries.md#5-validate-and-test-the-configuration

Your conf dir you are passing in is wrong.

**Reply by thiagojcb** [reply]
indeed... I was missing a `~/`
thanks a lot!
